test/test_manifest.json

```diff
@@ -755,6 +755,12 @@
        "rounds": 1,
        "type": "load"
     },
+    {  "id": "listbox_actions",
+       "file": "pdfs/listbox_actions.pdf",
+       "md5": "a8ec51fad4fe67509a33f142bc64c508",
+       "rounds": 1,
+       "type": "eq"
+    },
```

Comment:
Is a reference test actually necessary/useful here, especially given the added integration-tests, since the changes in this patch seem unlikely to be testable using a regular `eq` test?